job/replay_manager.go

```diff
@@ -236,6 +133,18 @@ func (m *Manager) spawnServiceWorker() {
 	}
 }
 
+// AirflowSyncer to sync for latest replay status
+func (m *Manager) AirflowSyncer() {
+	logger.I("start synchronizing replays...")
+	ctx, cancelCtx := context.WithTimeout(context.Background(), m.config.WorkerTimeout)
+	if err := m.replaySyncer.Sync(ctx, m.config.RunTimeout); err != nil {
+		logger.E(errors.Wrap(err, "syncer failed to process"))
+		cancelCtx()
+	}
+	logger.I("replays synced")
+	cancelCtx()
```

Comment:
We can `defer` this call before line 140 right to avoid calling it at 2 places.

Reply:
changed to use defer